App/src/main/webapp/style.css

```diff
@@ -1,4 +1,67 @@
 * {
-    margin: 0;
-    padding: 0;
+  margin: 0;
+  padding: 0;
+}
+
+/* color variables */
```

Comment:
Whoah since when does CSS have variables?? Times, they are a changing...

Reply:
Easily one of my favorite things in CSS, never looked at SASS since 👍